orttraining/orttraining/test/python/checkpoint/orttraining_test_load_checkpoint.py

```diff
@@ -0,0 +1,1472 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+################################################################################
+# Refer to orttraining_test_checkpoint.py for an overview about Checkpoint tests
+################################################################################
+
+import os
+import pickle
+from numpy.testing import assert_allclose
+import numpy as np
+import argparse
+import glob
+
+import torch
+import torch.distributed as dist
+
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import onnxruntime
+from onnxruntime.training import checkpoint
+from _test_helpers import distributed_setup, create_orttrainer_and_load_checkpoint, split_state_dict, aggregate_states
+
+global_fp16_fp32_atol = 1e-3
```

Comment:
You are only using this tolerance for fp16. When you omit `atol` from `assert_allclose` for fp32 tests, for example, `1e-7` is assumed

Let's merge like this, but in a future PR, let's either create a FP32_ATOL and FP16_ATOL or just rename this one to global_fp16_atol and use the 1e-7 for fp32

Reply:
So, ```global_fp16_fp32_atol``` atol is not for fp16 or fp32, but for comparison between fp16 and fp32. Any comparison between two fp16s is done using ```atol=1e-7``` which is the default value. Similarly, for all comparisons between two fp32s is again done using ```atol=1e-7```.

Reply:
Having said that, maybe it could have been explicitly set. Will update in the next pull request.

Reply:
I think it is a good name then. Let it be